lib/llvm-backend/src/backend.rs

```diff
@@ -176,23 +175,22 @@ impl LLVMBackend {
         _stackmaps: &StackmapRegistry,
         _module_info: &ModuleInfo,
         target_machine: &TargetMachine,
+        llvm_callbacks: &Option<Rc<RefCell<dyn LLVMCallbacks>>>,
```

Comment:
Might make sense to make this take a &mut directly, the fact that it's an `Rc<RefCell<>>` doesn't seem relevant to the function

Reply:
You should be able to have an `Option<&mut dyn LLVMCallbacks>` I believe

Reply:
Something like `val.map(|inner| inner.borrow_mut())` might do the trick

Reply:
Just marking down for review that Mark and I paired on this for a while and weren't able to make it work.

Reply:
Or that the cure seemed worse than the disease, so we chose to not bother